swift/ql/lib/codeql/swift/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -574,6 +574,14 @@ private module PostUpdateNodes {
 
     override DataFlowCallable getEnclosingCallable() { result = TDataFlowFunc(n.getScope()) }
   }
+
+  class SummaryPostUpdateNode extends SummaryNode, PostUpdateNodeImpl {
```

Comment:
Please could you explain the purpose of this class?

Reply:
This was a missing piece in the data flow library to catch cases like the this: https://github.com/github/codeql/pull/11027/files#diff-7e1f3d77d83b67247f8f21045e82539532beff203bfccf85a0080314064f3ffaR89

Basically, if a summary introduced a step involving updating the taint of a node (like `arg -> qualifier` flow), we weren't propagating flow correctly through it because of this missing node.

Reply:
Indeed, this change LGTM 👍.